test/boost/sstable_resharding_test.cc

```diff
@@ -144,9 +144,14 @@ SEASTAR_TEST_CASE(sstable_resharding_test) {
 }
 
 SEASTAR_TEST_CASE(sstable_resharding_over_s3_test) {
+    auto opts = make_test_object_storage_options();
```

Comment:
i'd propose something like:

```c++
static boost::test_tools::assertion_result
has_scylla_test_env(boost::unit_test::test_unit_id)
{
    if (::getenv("SCYLLA_TEST_ENV")) {
        return true;
    }
    boost::test_tools::assertion_result result{false};
    result.message()
        << "Test environment is not configured. "
        << "Check test/pylib/minio_server.py for an example of how to configure the environment for it to run."
    return result;
}
SEASTAR_TEST_CASE(sstable_resharding_over_s3_test,
                  *boost::unit_test::precondition(has_scylla_test_env) {
    // ...
}
```

see https://www.boost.org/doc/libs/1_82_0/libs/test/doc/html/boost_test/tests_organization/enabling.html#boost_test.tests_organization.enabling.runtime_run_status

Reply:
Interesting, let me try it out